Saule/Serialization/JsonApiContractResolver.cs

```diff
@@ -8,22 +9,47 @@ namespace Saule.Serialization
     internal class JsonApiContractResolver : DefaultContractResolver
```

Comment:
Looking back at this, I think it might be better to create a separate Contractor for the source serializations instead of modifying the original.